ui/tests/integration/components/auth-method/configuration-test.js

```diff
@@ -8,34 +8,34 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import AuthMethodResource from 'vault/resources/auth/method';
 
 module('Integration | Component | auth-method/configuration', function (hooks) {
   setupRenderingTest(hooks);
 
   hooks.beforeEach(function () {
     this.store = this.owner.lookup('service:store');
-    this.createModel = (path, type) => {
-      this.model = this.store.createRecord('auth-method', { path, type });
-      this.model.set('config', this.store.createRecord('mount-config'));
+    this.createMethod = (path, type) => {
+      this.method = new AuthMethodResource({ path, type, config: { listingVisibility: false } }, this);
```

Comment:
Should this match what is returned by the API, if so `listingVisibility` is a [string](https://developer.hashicorp.com/vault/api-docs/system/auth#listing_visibility-1)

Reply:
Yes good catch! The component is setup to return a boolean for unauth/hidden so I will update this.